### PR TITLE
Unhardcode name of state field.

### PR DESCRIPTION
**Pull request by ticosax**
Make it customizable through the FSM_STATE_FIELD_NAME class property

needs #4 for the tests to pass. I will rebase it after.

### CODE REVIEW

File: django_fsm_freeze/models.py [hunk 1]
```diff
@@ -15,12 +15,13 @@ class FreezableFSMModelMixin(DirtyFieldsMixin, models.Model):
     class Meta:
         abstract = True
 
-    FROZEN_IN_STATES = ()
-    NON_FROZEN_FIELDS = ('state',)
+    FROZEN_IN_STATES: tuple = ()
+    FSM_STATE_FIELD_NAME: str = 'state'
```
Comment (ming-tung) [inline django_fsm_freeze/models.py:19]: Is it possible to include a test to check if the configuration is correct? i.e. the value of `FSM_STATE_FIELD_NAME` is a field in the model and it's an `FSMField`?

File: README.md [hunk 2]
```diff
@@ -44,10 +44,13 @@ class MyDjangoFSMModel(FreezableFSMModelMixin):
     NON_FROZEN_FIELDS = FreezableFSMModelMixin.NON_FROZEN_FIELDS + (
         'a_mutable_field',
     )
+    FSM_STATE_FIELD_NAME = 'state'  # defaults to 'state'
```
Comment (ming-tung) [inline README.md:47]: I am thinking about the comment to mention the link between this value and the `FSMField`. Maybe something like:
```
# Assign this with the name of the`FSMField` if it is not 'state'. 
If your are using 'state' as the `FSMField` in your model, you can leave this one out.
See example in `mytest/models.py:FakeModel2`
```

**Comment by ming-tung** [comment]
Thanks for proposing the changes!

File: django_fsm_freeze/models.py [hunk 3]
```diff
@@ -30,11 +32,23 @@ def freeze_check(self) -> None:
...
         errors = defaultdict(list)
+        try:
+            fsm_state_field = cls._meta.get_field(cls.FSM_STATE_FIELD_NAME)
+        except FieldDoesNotExist:
+            errors[cls.FSM_STATE_FIELD_NAME].append(
+                f'{cls.FSM_STATE_FIELD_NAME!r} field does not exist.'
+            )
+        else:
+            if not isinstance(fsm_state_field, FSMField):
+                errors[cls.FSM_STATE_FIELD_NAME].append(
```
Comment (ming-tung) [inline django_fsm_freeze/models.py:43]: For the error message, `errors['FSM_STATE_FIELD_NAME'].append(....)`?

Comment (ticosax) [reply django_fsm_freeze/models.py:43]: makes sense

**Review by ming-tung — APPROVED** [review]
thanks!

**Pull request merged**